src/ca/mcgill/cs/jetuml/gui/GraphPanel.java

```diff
@@ -94,6 +93,7 @@
 	private static final Color GRABBER_FILL_COLOR = new Color(173, 193, 214);
 	private static final Color GRABBER_FILL_COLOR_TRANSPARENT = new Color(173, 193, 214, 75);
 	
+	private GraphFrame aFrame;

```

Comment:
doesn't this introduce a circular dependency?

Reply:
Once `GraphPanel` is migrated to JavaFX, the `GraphFrame` will be accessible through the object graph and the circular dependency will be removed.